overview/index.html

```diff
@@ -126,7 +128,7 @@
                         rowClass = "class='warning'";
                     }
 
-                    $("#items").append('<tr '+rowClass+' data-url="'+url+'"><td>'+shortName+'</td><td>'+ repoLink +'</td><td><a href="'+url+'">Link</a></td><td>'+item.labels["Git-SHA"]+'</td><td>'+sinceDuration+'</td><td>'+item.invocationCount+'</td><td>'+item.replicas+'</td></tr>');
+                    $("#items").append('<tr '+rowClass+' data-url="'+url+'"><td>'+shortName+'</td><td>'+ repoLink +'</td><td><a href="'+url+'">Link</a></td><td>'+item.labels["Git-SHA"].slice(0, 7)+'</td><td>'+sinceDuration+'</td><td>'+item.invocationCount+'</td><td>'+item.replicas+'</td></tr>');
```

Comment:
Slicing 0,7 probably is assuming too much. We can have `unknown` appear here too I think - perhaps we should check the exact length and if it matches a SHA then trim? What is the default GitHub uses in the overview page for SHAs?

Reply:
It also uses first 7 characters.